admin/access-control/organizations.md

```diff
@@ -312,13 +312,23 @@ organization.
 
 ## Namespaces
 
-Coder's helm chart includes `namespaceWhitelist`. This value takes a list of the
-namespaces that are in your cluster and are available to Coder. If you don't set
-`namespaceWhitelist`, the only available namespace is the one Coder has deployed
-to (this is also the default namespace).
+> **Notice**: The `namespaceWhitelist` field has been deprecated in Coder version
+1.17.
 
-If you're a site manager, you can assign a namespace to an organization; all
-resources for that organization now reside in that namespace. **Please note that
-this field is not editable once set.**
+Coder's helm chart previously included a `namespaceWhitelist` field, which took
+a list of namespaces in your cluster and available to Coder. This field has been
+removed in version 1.17 to support [Workspace
+Providers](../workspace-providers/index.md).
 
-![Organization Resources](../../assets/org-resources.png)
+This field is no longer used in the Coder helm chart, and any changes you make
+will not be applied to Coder deployments running version 1.17+. Existing
+environments can continue to function in the previously allowed namespaces, but
+you cannot create new environments in these namespaces.
```

Comment:
Changes to this value are not permitted unless no envs are left in removed namespaces for deployments migrating from 1.16. If you remove namespaces from this value then envs in those namespace become inaccessible